x-pack/legacy/plugins/infra/public/pages/shared/settings/index.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { InjectedIntl, injectI18n } from '@kbn/i18n/react';
+import React from 'react';
+import { UICapabilities } from 'ui/capabilities';
+import { injectUICapabilities } from 'ui/capabilities/react';
+import { SourceConfigurationSettings } from '../../../components/source_configuration/source_configuration_settings';
+
+interface SettingsPageProps {
+  intl: InjectedIntl;
+  uiCapabilities: UICapabilities;
+}
+
+export const SettingsPage = injectUICapabilities(
+  injectI18n(({ intl, uiCapabilities }: SettingsPageProps) => (
```

Comment:
Same as above, use `i18n.translate`. Although I don't actually see where `intl` is being used in this component?

Reply:
Will fix 👍